SparkPost/matchbox
# UX-612, UX-657 Convert Banner and Text components to TypeScript

### What Changed

- Converted `Banner` component to TypeScript
- Converted `Text` component to TypeScript
- Updated paths in tsconfig.json
- Added index.d.ts file to declare image and video files to fix lint error
- Update `precommit` command to work with typescript files

### How To Test or Verify

- Run locally and make sure everything works as expected and tests pass
- `npm run start:libby`

### PR Checklist
